Build fails due to missing symbol with clang-cl 13.0.0 on MSVC x64

I am trying to build oneTBB with clang-cl from LLVM 13.0.0 (current latest release) together with Visual Studio 2019 v16.11.5 (also current latest). I also use current latest cmake 3.21.4 and a very recent Ninja. hwloc is https://github.com/open-mpi/hwloc/releases/tag/hwloc-2.5.0.

Everything works quite nicely, until about 95%. Then I get the following error that does not mean much to me:
```
[build] FAILED: clang_13.0_cxx17_64_md_debug/tbbbind_2_5_debug.dll clang_13.0_cxx17_64_md_debug/tbbbind_2_5_debug.lib 
[build] cmd.exe /C "cd . && C:\data\Tools\bin\cmake.exe -E vs_link_dll --intdir=src\tbbbind\CMakeFiles\tbbbind_2_5.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100203~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100203~1.0\x64\mt.exe --manifests  -- C:\BDA\ci\LLVM-13.0.0\bin\lld-link.exe  src\tbbbind\CMakeFiles\tbbbind_2_5.dir\tbb_bind.cpp.obj  /out:clang_13.0_cxx17_64_md_debug\tbbbind_2_5_debug.dll /implib:clang_13.0_cxx17_64_md_debug\tbbbind_2_5_debug.lib /pdb:clang_13.0_cxx17_64_md_debug\tbbbind_2_5_debug.pdb /dll /version:3.4 /machine:x64   /DEBUG:FULL /ignore:4099 /debug /INCREMENTAL /DEF:C:/BDA/Src/oneTBB/src/tbbbind/def/win64-tbbbind.def  C:\data\Debug\lib\libhwloc.lib  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib  && cmd.exe /C "cd /D C:\data\Debug\oneTBB\src\tbbbind && C:\data\Tools\bin\cmake.exe -DBINARY_DIR=C:/data/Debug/oneTBB -DSOURCE_DIR=C:/BDA/Src/oneTBB -DBIN_PATH=C:/data/Debug/oneTBB/clang_13.0_cxx17_64_md_debug -DVARS_TEMPLATE=windows/env/vars.bat.in -DVARS_NAME=vars.bat -DTBB_INSTALL_VARS=OFF -DTBB_CMAKE_INSTALL_LIBDIR=lib -P C:/BDA/Src/oneTBB/integration/cmake/generate_vars.cmake""
[build] LINK Pass 1: command "C:\BDA\ci\LLVM-13.0.0\bin\lld-link.exe src\tbbbind\CMakeFiles\tbbbind_2_5.dir\tbb_bind.cpp.obj /out:clang_13.0_cxx17_64_md_debug\tbbbind_2_5_debug.dll /implib:clang_13.0_cxx17_64_md_debug\tbbbind_2_5_debug.lib /pdb:clang_13.0_cxx17_64_md_debug\tbbbind_2_5_debug.pdb /dll /version:3.4 /machine:x64 /DEBUG:FULL /ignore:4099 /debug /INCREMENTAL /DEF:C:/BDA/Src/oneTBB/src/tbbbind/def/win64-tbbbind.def C:\data\Debug\lib\libhwloc.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:src\tbbbind\CMakeFiles\tbbbind_2_5.dir/intermediate.manifest src\tbbbind\CMakeFiles\tbbbind_2_5.dir/manifest.res" failed (exit code 1) with the following output:
[build] lld-link: error: <root>: undefined symbol: global
[build] [40/41  97% :: 9.722] C:\BDA\ci\LLVM-13.0.0\bin\clang-cl.exe   -TP -DTBB_USE_DEBUG -D__TBBMALLOCPROXY_BUILD -Dtbbmalloc_proxy_EXPORTS -IC:\BDA\Src\oneTBB\src\tbbmalloc_proxy\..\..\include -IC:\BDA\Src\oneTBB\src\tbbmalloc\..\..\include /MDd /Z7 /arch:AVX2  /D_ITERATOR_DEBUG_LEVEL=0 /DWINVER=0x0603 /D_WIN32_WINNT=0x0603 /DDEBUG /D_CRT_SECURE_NO_DEPRECATE /D_SILENCE_ALL_CXX17_DEPRECATION_WARNINGS  -Wno-error=unused-but-set-variable   /DWIN32 /D_WINDOWS  /GR /EHsc /Zi /Ob0 /Od /RTC1 -MDd /W4 /wd4324 -D_CRT_SECURE_NO_WARNINGS /GS /volatile:iso /FS -mrtm -mwaitpkg -std:c++17 /showIncludes /Fosrc\tbbmalloc_proxy\CMakeFiles\tbbmalloc_proxy.dir\proxy.cpp.obj /Fdsrc\tbbmalloc_proxy\CMakeFiles\tbbmalloc_proxy.dir\ -c -- C:\BDA\Src\oneTBB\src\tbbmalloc_proxy\proxy.cpp
[build] ninja: build stopped: subcommand failed.
```

--- a/src/tbbbind/def/win32-tbbbind.def
+++ b/src/tbbbind/def/win32-tbbbind.def
@@ -14,7 +14,6 @@
 
 EXPORTS
 
-global
 __TBB_internal_initialize_system_topology
 __TBB_internal_apply_affinity
 __TBB_internal_restore_affinity

--- a/src/tbbbind/def/win64-tbbbind.def
+++ b/src/tbbbind/def/win64-tbbbind.def
@@ -14,7 +14,6 @@
 
 EXPORTS
 
-global
 __TBB_internal_initialize_system_topology
 __TBB_internal_apply_affinity
 __TBB_internal_restore_affinity

